cmd/context/utils.go

```diff
@@ -186,12 +186,43 @@ func LoadStackWithContext(ctx context.Context, name, namespace string, stackPath
 	return s, nil
 }
 
+//LoadManifestV2WithContext initializes the okteto context taking into account command flags and manifest namespace/context fields
+func LoadManifestV2WithContext(ctx context.Context, namespace, path string) error {
```

Comment:
A utils dir and context package are not a great choice of package or filenames as it is a sort of grab bag and lacks focus. The package is probably what I care about most as it doesn't seem like good isolation to put all sorts of things that obtain info from context in here but should focus more on something like `manifest`. I don't think we should fix that package as part of this commit but we could stop growing it. I think this function might fit better in a package that's focused on the actual manifest or possibly a slightly less focused package related to general actions of getting, parsing and checking various config files that okteto uses?